src/components/Card/WCard.stories.ts

```diff
@@ -0,0 +1,65 @@
+import type { Meta, StoryObj } from '@storybook/vue3'
+import WCard from './WCard.vue'
+import { CardType } from './WCard'
+
+const meta: any = {
+  title: 'Example/WCard',
+  component: WCard,
+  tags: ['autodocs'],
+  argTypes: {
+    type: { control: 'select', options: ['base', 'icon', 'link'] }
+  },
+  args: {
+    type: CardType.Base
+  }
+} satisfies Meta<typeof WCard>
+
+export default meta
+type Story = StoryObj<typeof meta>
+
+export const Base: Story = {
+  render: (args: any) => ({
+    components: { WCard },
+    setup() {
+      return { args }
+    },
+    template: `<WCard v-bind="args"><template #content>Link description</template></WCard>`
```

Comment:
Could we replace this text with something with a lenght similar to the one we are going to use i.e in the website? 
```suggestion
    template: `<WCard v-bind="args"><template #content>Lorem ipsum dolor sit amet, consectetuer adipiscing elit. Aenean commodo ligula eget dolor. Aenean massa. Cum sociis natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus.</template></WCard>`
```